test/topics-list.test.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable */
+
+import { fixture, html, expect } from '@open-wc/testing';
+import { spy, assert } from 'sinon';
+import { TopicsList } from '../src/components/topics-list';
+
+describe('Topics list test', () => {
+  let element;
+  let topics;
+
+  beforeEach(async () => {
+    element = await fixture(html` <div id="topics"></div> `);
+    topics = new TopicsList('topics', localStorage);
+  });
+
+  it('constructor is run correctly', async () => {
+    expect(topics.target).equals('topics');
+    expect(topics.domTarget).not.equals(undefined);
+    expect(topics.storage).equals(localStorage);
+    expect(topics.storageName).equals('completedTopics');
+    expect(typeof topics.allTopics).equals('object');
+    expect(topics.allTopics.length).not.equals(0);
+  });
+
+  it('the output of the topics is rendered as expected', async () => {
+    const articles = element.querySelectorAll('article');
+    expect(articles.length).equals(11);
+    expect(articles[0].innerText).equals('Skills and knowledge');
+    expect(articles[0].innerHTML.includes('div class="content"')).equals(true);
+
+    expect(articles[0].innerHTML.includes('img src="')).equals(true);
+    expect(articles[0].innerHTML.includes('alt="Skills and knowledge"')).equals(
+      true,
+    );
+    expect(articles[0].innerHTML.includes('button')).equals(true);
+
+    expect(articles[1].innerText).equals('Health and hygiene');
+    expect(articles[1].innerHTML.includes('div class="content"')).equals(true);
+    expect(articles[1].innerHTML.includes('img src="')).equals(true);
+    expect(articles[1].innerHTML.includes('alt="Health and hygiene"')).equals(
+      true,
+    );
+    expect(articles[1].innerHTML.includes('button')).equals(true);
+
+    expect(articles[2].innerText).equals('Recieve food');
```

Comment:
Misspelled "Receive"

Reply:
Tests don't mind what is there as long as it matches.

Reply:
Yes, but that means the text visible to end users is also misspelled.